Makefile

```diff
@@ -381,6 +381,18 @@ install-web-tools: build/toolchain/bin/hugo$(EXE_EXTENSION) build/toolchain/bin/
 install-protoc-tools: build/toolchain/bin/protoc$(EXE_EXTENSION) build/toolchain/bin/protoc-gen-go$(EXE_EXTENSION) build/toolchain/bin/protoc-gen-grpc-gateway$(EXE_EXTENSION) build/toolchain/bin/protoc-gen-swagger$(EXE_EXTENSION)
 install-openmatch-tools: build/toolchain/bin/certgen$(EXE_EXTENSION)
 
+
+install-stress-test-tools:
```

Comment:
Add the deleted code from https://github.com/GoogleCloudPlatform/open-match/commit/4cbee9d8a7f5ccbbee15222e56fbde176fc3710c#diff-b67911656ef5d18c4ae36cb6741b7965 back to replace this target. This way it's more cross platform. We purposely do not use apt-get in the makefile because that not going to work on Windows and Mac. It's ok to require users to download Python and I believe the instructions do already.

Using the virtualenv in the build directory allows us to have our own isolated environment of Python that does not depend on the user's environment and pollute their default configuration.